phoenix-core-server/src/main/java/org/apache/phoenix/coprocessor/tasks/CdcStreamPartitionMetadataTask.java

```diff
@@ -136,13 +136,16 @@ private void upsertPartitionMetadata(PhoenixConnection pconn, String tableName,
             throws SQLException {
         try (PreparedStatement ps = pconn.prepareStatement(CDC_STREAM_PARTITION_UPSERT_SQL)) {
             for (HRegionLocation tableRegion : tableRegions) {
+                // set parent_partition_id, partition_end_time to null
                 RegionInfo ri = tableRegion.getRegionInfo();
```

Comment:
nit: this is deprecated, alternative: `RegionInfo ri = tableRegion.getRegion()`

Reply:
done